Misc/NEWS.d/next/Library/2018-12-27-19-23-00.bpo-35568.PutiOC.rst

```diff
@@ -0,0 +1 @@
+Expose :c:func:`raise(signum)` as `raise_signal`
```

Comment:
Someone may know better, but I'm pretty sure the `:c:func:` directive is only for our own C functions, which means this will end up being a broken link.

Reply:
I've used [this comment](https://github.com/python/cpython/pull/11248/files/eb1127b1798a420320f1555c3cff8c5aa859d6f4#diff-76ae9c0ac270677533162071d84bb2a8) where `:c:func:` was used for `GetFinalPathNameByHandle` as an example. If this is not correct - I'd be happy to fix it.

Reply:
@zooba's right, `:c:func:` is used for CPython C API functions.  It's a bug if it is used for Posix/etc functions in other places.

Reply:
Should it then be just a `:func:`? It is a bit confusing, [Documenting Python](https://devguide.python.org/documenting/) doc only mentions `c:func` as :
>c:func
The name of a C-language function. Should include trailing parentheses.

and  I can see a number places in existing docs where externally defined  functions are referenced via `c:func:`, i.e.
```
*datetime.rst*

if the timestamp is out
   of the range of values supported by the platform C :c:func:`localtime` function
```
```
*asyncio-subprocess.rst*

On Windows the Win32 API function :c:func:`TerminateProcess` is
   called to stop the child process.
```
```
*asyncore.rst*

If your operating system supports the :c:func:`select` system call in its I/O
library (and nearly all do),

```
```
*ctypes.rst*

I will present an example here which uses the standard C library's
:c:func:`qsort` function
```

Reply:
`:c:func:qsort` creates a link to `.. c:function:: qsort` block.
If the block is absent no link is created but `:c:func:qsort` is pointless in this case.
Since CPython documentation has no definition for `.. c:function:: qsort` the cross-reference should be removed, consider it as a hurtless bug.

The same is for `raise`. Double backticks should work fine